content/en/continuous_delivery/deployments/ciproviders.md

```diff
@@ -123,6 +123,10 @@ the `DD_GITHUB_JOB_NAME` environment variable needs to be exposed, pointing to t
         - run: datadog-ci deployment mark ...
     ```
 
+## Correlate deployments with CI pipelines
+
+This is done by default so there is no action needed. See the section ["visualize correlated pipelines and deployments"][18] to see how the visualization looks in Datadog.
```

Comment:
```suggestion
By default, deployments are correlated with CI pipelines. See the [Visualize correlated pipelines and deployments][15] section on the CD Visibility page for an example.
```

Reply:
I'll add the link, since GitHub will not allow me to make a suggestion on that section

Reply:
Thanks! I copy pasted it and did not notice